style.css

```diff
@@ -924,6 +930,7 @@
   .nav-desktop a {
     background-color: #ffff;
     text-decoration: none;
+    color: #344563;
   }
 
   .card-desktop {
```

Comment:
[OPTIONAL] Great job fixing the desktop version of your cards as requested by the previous reviewer 💯 . However, there still seems to be an issue with your cards in the mobile version. There are still not centered, and there seems to be some extra margin on the right:

Your design: 
![margin](https://user-images.githubusercontent.com/111214189/229267559-02109858-e9db-459e-b06a-aac740fae1db.png)

Figma template:
![figma](https://user-images.githubusercontent.com/111214189/229267663-92e5c7d4-7186-4852-bca5-b86099b2a68d.png)

Kindly fix these margins to fit the template and make sure that your desktop version still displays correctly. This issue was missed by the previous reviewer but I **highly** recommend you implement it. This will make your portfolio look more professional and improve the user experience with your website.

Reply:
I think i have, hope so 
![image](https://user-images.githubusercontent.com/111823606/229289013-830a0806-54d0-4b41-9dd0-2e65be45c6dd.png)